src/main/java/org/truffleruby/interop/OutgoingForeignCallNode.java

```diff
@@ -432,12 +463,20 @@ public InvokeOutgoingNode(String name, int argsLength) {
         public Object executeCall(VirtualFrame frame, TruffleObject receiver, Object[] args) {
             assert args.length == argsLength;
 
+            final Object foreign;
+
             try {
-                return ForeignAccess.sendInvoke(node, receiver, name, args);
+                foreign = ForeignAccess.sendInvoke(
+                        node,
+                        receiver,
+                        name,
```

Comment:
Should the `name` be converted to foreign here?

Reply:
It's already a Java string though.

Reply:
Where is it converted or where is it typed as a Java String?

Reply:
It's like `a.foo` - the parser gives us `foo` as a Java string. Look a couple of lives above to the constructor.

Reply:
I see, I missed it somehow and the GitHub diff cuts right before it.